packages/components/src/button/index.js

```diff
@@ -137,7 +137,7 @@ export function Button( props, ref ) {
 				{ element }
 				{ describedBy && (
 					<VisuallyHidden>
-						<span id={ descriptionId }>{ describedBy }</span>
+						<span id={ describedById }>{ describedBy }</span>
```

Comment:
I was trying to make the aria-describedby id match the id on the element. The way it was before, aria-describedby would take one id and the element would end up with the unique id from lodash. At least that's how the code looked to me, called it a bug fix.

Reply:
The  way  I understand it is that if  you pass `describedBy` you shouldn't be passing `aria-describedby` to the button because you're telling the "Button" component to render the description itself (os it just generates its own id) while if you pass `aria-describedby` it's the opposite: you render the description outside of the component and you just pass the id.

So I think the right approach here, would be to restore `descriptionId` here and  drop the passed `aria-describedby` 

That said, I agree that it's not clear at first and  it's probably something that should be documented better in the `Button` component README.

Reply:
Do you think the ID will always be unique even with a bunch of blocks? I reverted my change and let it go back to using the randomized ID.

Reply:
yes, lodash.uniqueId ensure that it's a unique id per component instance.